ExampleMod/Projectiles/ExampleLastPrismHoldout.cs

```diff
@@ -0,0 +1,224 @@
+﻿using ExampleMod.Items.Weapons;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+using static Terraria.ModLoader.ModContent;
+
+namespace ExampleMod.Projectiles
+{
+	public class ExampleLastPrismHoldout : ModProjectile
```

Comment:
All the same suggestions count for this class. Just see what you can/feel like improve.